README.md

```diff
@@ -13,7 +21,7 @@ funcpack pack ./
 
 You can then test locally using the CLI tool: `func run <myfunc>`
 
-When uploading your files, you need to include your `package.json`, but you don't need your `node_modules` directory.
+When uploading your files, you need to include the single `.funcpack` directory (in the Functions App root), but you don't need your `node_modules` directory for each Function.
```

Comment:
PS did you look at tree shaking? It's still a bit experimental in webpack 2 but could help. I guess how much effort you put in depends on how long before a file systems fix lands :)